### ISSUE TITLE
Add option to Reset the Selector

### ISSUE THREAD

**Issue by raeesnazeer** [andreskiu/image_selector_formfield#3]
There is no option to clear the image after selection like the textfield controller clear option.

**Comment by andreskiu** [comment]
Yes, it's a debt i have... I will make as soon as posible but... i can take a while. If you want to contribute... PR are accepted
